src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelDelegateCommands.cs

```diff
@@ -73,6 +73,8 @@ private void InitializeDelegateCommands()
             SetNumberFormatCommand = new DelegateCommand(SetNumberFormat, CanSetNumberFormat);
             GetBranchVisualizationCommand = new DelegateCommand(GetBranchVisualization, CanGetBranchVisualization);
             DumpLibraryToXmlCommand = new DelegateCommand(model.DumpLibraryToXml, model.CanDumpLibraryToXml);
+            ShowGalleryCommand = new DelegateCommand(ShowGallery, CanShowGallery);
+            CloseGalleryCommand = new DelegateCommand(CloseGallery, CanCloseGallery);
```

Comment:
noted, we need to use ```o => true``` since the function signatures expect object as the parameter